### ISSUE TITLE
Neutered symptom on virus changes after placing in storage

### ISSUE THREAD

**Issue by JannisKuckei** [tgstation/tgstation#31271]
[Directions]: # (Include the Round ID from the Status panel or retrieve it from https://atlantaned.space/newSS13tools/round.php ! If you believe the issue to be caused by a testmerge [OOC tab -> Show Server Revision], report it in the pull request's comment section instead. Explain your issue in detail, including the steps to reproduce it.)

[For Admins]: # (Oddities induced by var-edits and other admin tools are not necessarily bugs. Verify that your issues occur under regular circumstances before reporting them.)

Round ID: 76940

I had a beaker with a virus that had the symptoms Eternal Youth and Fever. I added 1u of that and 1u of Formaldehyde to another beaker to neuter Fever. I put it in the Pandemic 2200 and saw that fever was neutered. Then I created a virus culture bottle of it and placed it in the smart virus storage. I had the same issue yesterday, so I immediately took the bottle out again in suspicion and put it in the Pandemic 2200, and lo and behold, the neutered symptom was now Eternal Youth while Fever was active.

Hope this is not a feature I'm not understanding instead of a bug!

It's probably worth noting I have only had this happen with fever,
Further edit: It just happened with Itching and Eternal Youth. This time, the culture bottle was placed on a table instead of the smart storage.

**Comment by Adanps** [comment]
I ran into this problem yesterday. Neutered Alkali perspiration, made a culture bottle with the pandemic, and sneezing (which was at the top of the symptom list) was neutered in the bottle instead when I checked it with the Pandemic. Haven't tested to see if the symptom that the Pandemic says is neutered lines up with what happens when you actually infect people with the virus. I imagine this is an issue worth looking into, since it won't be fun if virologists accidentally kill people with symptoms they thought were neutered.
